Upgrade from v10 to v11 (#191)
projectsyn/component-keycloak: docs/modules/ROOT/pages/how-tos/upgrade-10.x-to-11.x.adoc

= Upgrade from v10 to v11

This guide describes the steps to perform an upgrade of the component from version v9 to v10.

== Changes

> * The component requires at least a Kubernetes version of v1.21.

Reviewer: ```suggestion
* The component requires Kubernetes  v1.21 or newer.
```

> * K8up v1 support dropped.

Reviewer: ```suggestion
* K8up v1 support has been removed.
```
